app/models/reviewable_queued_post.rb

```diff
@@ -163,16 +163,22 @@ def perform_reject_post(performed_by, args)
 
   def perform_revise_and_reject_post(performed_by, args)
     pm_translation_args = {
-      topic_title: self.topic.title,
-      topic_url: self.topic.url,
+      topic_title: self.topic.blank? ? self.payload["title"] : self.topic.title,
```

Comment:
```suggestion
self.topic&.title || self.payload["title"]
```

Ultra minor. Suggested because we are using & for url so it will look a bit more consistent

Reply:
Ah nice that will be good, ty :)